docs/releasenotes.md

```diff
@@ -7,7 +7,30 @@ title: Haal Centraal BRP Personen Bevragen
 
 ## Versie 2.0.0
 
-De property reisdocumenten is verwijderd uit "Persoon".
+- het request is gewijzigd van GET naar POST, waarbij alle parameters verplaatst zijn van de querystring in de url naar de requestbody
+- requestbody is polymorf gedefinieerd, waarbij in parameter 'type' bepaalt welke parameters gebruikt mogen en moeten worden
+- parameter 'fields' is verplicht (required)
+- in het antwoord zitten geen json-HAL elementen \_links en \_embedded
+- in het antwoord is veld 'ingeschrevenPersonen' hernoemd naar 'personen'
+- het antwoord op elke zoekvraag (elk request anders dan raadplegen op burgerservicenummer) bevat maximaal een beperkte set gegevens van de persoon
+- veld 'reisdocumenten' is verwijderd
+- alle datums zijn polymorf gedefinieerd: nieuw veld type geeft aan welke velden voor de datum verwacht kunnen worden
+- alle datums hebben nieuw veld 'langFormaat', die de datum levert in voor mensen leesbare vorm
+- alle enumeraties zijn omgezet naar Waardetabel type, met 'code' plus 'omschrijving'. Zie https://github.com/VNG-Realisatie/Haal-Centraal-BRP-tabellen-bevragen/blob/master/docs/tabelwaarden.csv voor mogelijke waarden
+- veld 'geslachtsaanduiding' is hernoemd naar 'geslacht'
+- groep 'kiesrecht' is gesplitst in 'uitsluitingKiesrecht' en 'europeesKiesrecht'
+- ook afgeleide gegevens (zoals aanschrijfwijze en leeftijd) zijn opgenomen in 'inOnderzoek'
+- naamvelden 'aanschrijfwijze', 'aanhef' en 'gebruikInLopendeTekst' zijn verplaatst naar groep 'adressering'
+- adresregels zijn verplaatst van groep 'verblijfplaats' naar groep 'adressering'
+- velden 'gemeenteVanInschrijving' en 'datumInschrijvingInGemeente' zijn verplaatst van groep 'verblijfplaats' naar het hoogste niveau ('personen')
+- velden van de immigratie zijn verplaatst van groep 'verblijfplaats' naar groep 'immigratie'
+- velden 'indicatieCurateleRegister' en 'indicatieGezagMinderjarige' zijn verplaatst van groep 'gezagsverhouding' naar het hoogste niveau ('personen')
+- de adresvelden van een verblijfplaats zijn gegroepeerd in verblijfplaats groep 'verblijfadres'
+- groep 'verblijfplaats' is polymorf gedefinieerd, waarbij type aangeeft of het een binnenlands adres, locatie, buitenlands adres of onbekende verblijfplaats is
+- groep 'nationaliteiten' is polymorf gedefinieerd, waarbij type aangeeft of het een nationaliteit, bijzonder Nederlanderschap, staatloosheid of onbekende nationaliteit betreft
+- wanneer er geen actuele partner is (zonder ontbinding van het huwelijk of partnerschap), wordt indien van toepassing het laatst ontbonden huwelijk of partnerschap geleverd, inclusief de datum van de ontbinding
+- wanneer de persoonsgegevens zijn geregistreerd door een RNI deelnemer, worden gegevens daarover opgenomen in groep 'rni'
+- wanneer voor een niet ingezetene (persoon in RNI) de gegevens zijn geverifieerd, worden gegevens daarover opgenomen in groep 'verificatie' 
```

Comment:
Is het relevant om te vermelden dat we geen verwijzingen naar andere repositories (common) meer opnemen ? Zo ja, dan moet dat nog toegevoegd worden.

Reply:
ik denk het niet. @MelvLee?

Reply:
Nee, is niet relevant. Dat er geen verwijzingen naar andere repositories zijn, heeft te maken met het vergemakkelijken van de beheer van de OAS specificaties.